spec/rails_helper.rb

```diff
@@ -64,4 +65,6 @@
 
   # 各テストコードから、FactoryBotという記述を省略可能にする
   config.include FactoryBot::Syntax::Methods
+
+  # config.include Sorcery::TestHelpers::Rails::System
```

Comment:
何か対応予定であれば、TODOを残した方が分かりやすいですね。
```suggestion
  # TODO: xxの理由でxxできない
  # config.include Sorcery::TestHelpers::Rails::System
```

Reply:
こちら 9ddc5bd で削除しました。